src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+
+import React, { useContext } from 'react';
+import { TodosContext, TodosControlContext } from '../context/TodosContext';
+import { useError } from '../context/ErrorContext';
+import { TodoError } from '../../types/enums';
+
+export const TodoForm: React.FC = () => {
+  const { addTodo, setInputTodo } = useContext(TodosControlContext);
+  const { inputTodo } = useContext(TodosContext);
```

Comment:
You don't need to store `inputTodo` in context as you need it only in the form, so store it locally in the component